lib/tasks/email_notes_to_admins.rake

```diff
@@ -0,0 +1,20 @@
+desc 'Send email to admins on reservations with notes'
+task email_notes_to_admins: :environment do
+  # gets all reservations with notes and sends an email to the admin of the
+  # application, to alert them.
+  notes_reservations_out = Reservation.has_notes.checked_out.notes_unsent
+  notes_reservations_in = Reservation.has_notes.checked_in.notes_unsent
+  Rails.logger.info "Found #{notes_reservations_out.size} reservations "\
+                    "checked out with notes and #{notes_reservations_in.size} "\
+                    'reservations checked in with notes. Sending a reminder '\
+                    'email...'
+  unless notes_reservations_out.empty? && notes_reservations_in.empty?
```

Comment:
See, here's a case where I think a guard clause would be relevant. I'd stick a

```
return if notes_reservations_out.empty? && notes_reservations_in.empty?
```

after the first logger call, since if those are both empty there's nothing else for us to do. I'd also add a second logger call *after* the guard clause to write `Sending a reminder email...` (taking that out of the first call). This way, we log that the task ran (since it will just say zero for both if the two arrays are empty) and then only run the email tasks / DB calls if there are relevant reservations. How does that sound?